src/crimson/common/logclient.cc

```diff
@@ -291,18 +293,19 @@ ceph::ref_t<Message> LogClient::_get_mon_log_message()
     ++p;
   }
   
-  return ceph::make_message<MLog>(m_fsid,
+  return crimson::make_message<MLog>(m_fsid,
 				  std::move(o));
 }
 
-void LogClient::_send_to_mon()
+seastar::future<> LogClient::_send_to_mon()
 {
   ceph_assert(is_mon);
   ceph_assert(messenger->get_myname().is_mon());
   ldout(cct,10) << __func__ << " log to self" << dendl;
   auto log = _get_mon_log_message();
   //TODO: implement crimson counterpart(WIP)
   //messenger->get_loopback_connection()->send_message2(std::move(log));
+  return m_monc->send_message(std::move(log));
```

Comment:
I think this is just a fixup, so seems worth squashing with the commit that introduced it.